lamindb/examples/cellxgene.py

```diff
@@ -0,0 +1,246 @@
+from typing import Collection, Literal
+
+import pandas as pd
+from anndata import AnnData
+from lamindb_setup.core.upath import UPath
+
+from lamindb.base.types import FieldAttr
+from lamindb.models import Feature, Schema, SQLRecord, ULabel
+from lamindb.models._from_values import _format_values
+
+CELLxGENESchemaVersions = Literal["4.0.0", "5.0.0", "5.1.0", "5.2.0", "5.3.0"]
+FieldType = Literal["ontology_id", "name"]
+
+
+def add_cxg_defaults_to_obs(adata: AnnData) -> None:
+    """Add missing default columns and values to the `obs` slot."""
+    cxg_categoricals_defaults = {
+        "cell_type": "unknown",
+        "development_stage": "unknown",
+        "disease": "normal",
+        "donor_id": "unknown",
+        "self_reported_ethnicity": "unknown",
+        "sex": "unknown",
+        "suspension_type": "cell",
+        "tissue_type": "tissue",
+    }
+    for name, default in cxg_categoricals_defaults.items():
+        if (
+            name not in adata.obs.columns
+            and f"{name}_ontology_term_id" not in adata.obs.columns
+        ):
+            adata.obs[name] = default
+
+
+def save_cxg_defaults() -> None:
```

Comment:
Not sure the UX of this function, there's no hint that users should call this function, right?

It'd be good to have it automatically checked in `get_cxg_schema`, if these labels are not added, add them, otherwise don't do anything. This can be a private function.

Reply:
I think not integrating is better because `save_...` should be called one time and `get_...` can be called many times.

So, I think it's good as is.